targetcli/ui_root.py

```diff
@@ -99,20 +98,20 @@ def ui_command_saveconfig(self, savefile=default_save_file):
 
                     if backup_error == None:
                         # Kill excess backups
-                        try:
-                            with open(universal_prefs_file) as prefs:
-                                backups = [line for line in prefs.read().splitlines() if re.match('^kept_backups\s*=', line)]
-                                kept_backups = int(backups[0].split('=')[1].strip())
-                        except:
-                            kept_backups = default_kept_backups
-
-                        files_to_unlink = list(reversed(backed_files_list))[kept_backups:]
+                        max_backup_files = int(self.shell.prefs['max_backup_files'])
+
+                        with open(universal_prefs_file) as prefs:
```

Comment:
I don't think we can get rid of the `try` here, unless we check that the "universal_prefs_file" exists and that we can successfully parse the backups array (which could be empty).

Reply:
+1 @tjakob 
I will submit the change soon.